app.py

```diff
@@ -1,21 +1,25 @@
 from flask import Flask, send_from_directory, request, jsonify
 from utils.data import get_criteria, get_pets
+from utils.ahp import AnalyticalHierarchyProcess
```

Comment:
I have just realised that the method is called `Analytic Hierarchy Process`, not `Analytical...`

Reply:
Huh, I saw analytical when I've read about on the internet, but in the book it's Analytic so I'll change that